controller/notifications/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright © 2024, F2FINTECH. ALL RIGHTS RESERVED.
+ *
+ * This software is the confidential information of F2FINTECH., and is licensed as
+ * restricted rights software. The use, reproduction, or disclosure of this software is subject to
+ * restrictions set forth in your license agreement with F2 FINTECH.
+ */
+const NotificationModel = require("../../model/notifications");
+
+const NotificationController = {
+  getNotifications: async (req, res) => {
```

Comment:
use Promise in your apis, we have used Promise in all other apis, so we should not change the format,
we should always follow 1 format to make our code more understandable, 
use promise here and remove async await.
Also remove async await from loan_provider apis